Formula/ballerina.rb

```diff
@@ -28,6 +28,8 @@ def install
 
     bin.install "bin/ballerina", "bin/composer"
     libexec.install Dir["*"]
+    #link tooling-plugin jar for the vscode
```

Comment:
Can you please explain this? CC @DomT4

Reply:
Can remove that comment. Added below;
`lib.install_symlink libexec/"lib/resources" => "resources"` to create a symlink for the resources folder.

Reply:
I don't think it'd be especially difficult for [upstream](https://github.com/ballerina-platform/ballerina-lang/tree/master/tool-plugins/vscode) to add a `ballerina.resources` variable to resolve this? Might be worth asking them, but this is harmless enough that I don't have an objection to it personally.

Reply:
+1 for **symlink** since introducing yet another `ballerina.resources` is an overkill for the end user.